learn2track/models/gru_regression.py

```diff
@@ -88,7 +102,7 @@ def get_output(self, X):
         self.regression_out = T.transpose(results[-1], axes=(1, 0, 2))
         return self.regression_out
 
-    def seq_next(self, x_t, subject_ids=None):
+    def seq_next(self, x_t, subject_ids=None, previous_directions=None):
```

Comment:
We should really drop this, no? Did you maintain it only for backward compatibility with the old track.py script?